mantidproject/mantid
# Fix IN5 pixel type's centre-of-bottom-base

This PR fixes the swapped y and z values of IN5's pixels type's `centre-of-bottom-base`.

**Report to:** [nobody].

**To test:**

Core review. Check that the `centre-of-bottom-base` z and y values are swapped.

Fixes #24820.


*This does not require release notes* because this only a cosmetic change. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
